app/pages/settings/profile.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import ImageSelector from '../../components/image-selector';
+import putFile from '../../lib/put-file';
+
+const MAX_AVATAR_SIZE = 65536;
+const MAX_HEADER_SIZE = 256000;
+
+class CustomiseProfile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      avatar: {},
```

Comment:
Uses empty objects for defaults, to avoid `undefined property of null` errors later.

Reply:
Is that a better approach in general?

Reply:
Depends on your code.